### PR TITLE
CLN/STY: Nitpicks

### PR DESCRIPTION
**Pull request by ShaharNaveh**
Things I find myself doing in multiple branches

### CODE REVIEW

File: pandas/_libs/index.pyx [hunk 1]
```diff
@@ -454,7 +463,7 @@ cdef class DatetimeEngine(Int64Engine):
 
 cdef class TimedeltaEngine(DatetimeEngine):
 
-    cdef _get_box_dtype(self):
+    cdef str _get_box_dtype(self):
         return 'm8[ns]'
```
Comment (jbrockmendel) [inline pandas/_libs/index.pyx]: not gonna change to double-quotes while you're here?

**Comment by jbrockmendel** [comment]
Looks fine.

Brainstorming ideas for expanding the range of what you work on: when editing a file, consider searching for TODO/FIXME comments and seeing if any of them are things you can address. (best case is right at the edge of what you're comfortable with).  E.g. there is one TODO in index.pyx that looks doable.

**Review by WillAyd — APPROVED** [review]
lgtm

File: pandas/_libs/algos.pyx [hunk 2]
```diff
@@ -791,8 +798,13 @@ ctypedef fused rank_t:
 
 @cython.wraparound(False)
 @cython.boundscheck(False)
-def rank_1d(rank_t[:] in_arr, ties_method='average',
-            ascending=True, na_option='keep', pct=False):
+def rank_1d(
+    rank_t[:] in_arr,
+    ties_method="average",
+    bint ascending=True,
```
Comment (MomIsBestFriend) [inline pandas/_libs/algos.pyx]: Added ```bint```

File: pandas/_libs/algos.pyx [hunk 3]
```diff
@@ -791,8 +798,13 @@ ctypedef fused rank_t:
...
+    na_option="keep",
+    bint pct=False,
```
Comment (MomIsBestFriend) [inline pandas/_libs/algos.pyx]: Added ```bint```

File: pandas/_libs/algos.pyx [hunk 4]
```diff
@@ -1009,8 +1021,14 @@ def rank_1d(rank_t[:] in_arr, ties_method='average',
         return ranks
 
 
-def rank_2d(rank_t[:, :] in_arr, axis=0, ties_method='average',
-            ascending=True, na_option='keep', pct=False):
+def rank_2d(
+    rank_t[:, :] in_arr,
+    int axis=0,
```
Comment (MomIsBestFriend) [inline pandas/_libs/algos.pyx]: added ```int```

File: pandas/_libs/algos.pyx [hunk 5]
```diff
@@ -1009,8 +1021,14 @@ def rank_1d(rank_t[:] in_arr, ties_method='average',
...
+    ties_method="average",
+    bint ascending=True,
```
Comment (MomIsBestFriend) [inline pandas/_libs/algos.pyx]: added ```bint```

File: pandas/_libs/algos.pyx [hunk 6]
```diff
@@ -1009,8 +1021,14 @@ def rank_1d(rank_t[:] in_arr, ties_method='average',
...
+    na_option="keep",
+    bint pct=False,
```
Comment (MomIsBestFriend) [inline pandas/_libs/algos.pyx]: added ```bint```

**Comment by MomIsBestFriend** [comment]
I forgot to mention the changes I've made which are not related to styling.

@jbrockmendel @WillAyd Can you please re-review?

**Comment by jorisvandenbossche** [comment]
Personally, I would stop doing such styling changes to cython (to resemble what black does) until there is black support for cython at which point we can do it in one go and consistently.

**Comment by MomIsBestFriend** [comment]
> Personally, I would stop doing such styling changes to cython (to resemble what black does) until there is black support for cython at which point we can do it in one go and consistently.

Noted.

**Comment by jreback** [comment]
thanks @MomIsBestFriend 

yeah let's hold off on cython changes for style until we have an auto formatter, however typing / fixing things are all ok.

**Pull request merged**